core/src/com/po/fuck/view/Sprites/BasicSpriteInfo.java

```diff
@@ -0,0 +1,38 @@
+package com.po.fuck.view.Sprites;
+
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.utils.JsonValue;
+
+/*
+ * assetsData.json description
+ * all textures have:
+ * each .png stores some table of frames for the object
+ * path - path to texture
```

Comment:
`path to THE texture`, also you have added `the` before every other variable, but not before this one